Tribler/Test/Core/Modules/RestApi/test_my_channel_endpoints.py

```diff
@@ -42,6 +35,26 @@ def create_my_channel(self, name, description):
         self.channel_db_handler.on_channel_from_dispersy('fakedispersyid', None, name, description)
         return self.channel_db_handler.getMyChannelId()
 
+    def create_fake_channel(self, channel_name, channel_description):
+        # Use a fake ChannelCommunity object (we don't actually want to create a Dispersy community)
+        my_channel_id = self.create_my_channel(channel_name, channel_description)
+        self.session.lm.channel_manager = ChannelManager(self.session)
+
+        channel_obj = ChannelObject(self.session, ChannelCommunityMock(my_channel_id))
+        self.session.lm.channel_manager._channel_list.append(channel_obj)
+        return my_channel_id
+
+
+class TestMyChannelEndpoints(AbstractTestMyChannelEndpoints):
+
+    @deferred(timeout=10)
+    def test_my_channel_unknown_endpoint(self):
+        """
+        Testing whether the API returns an error if an unknown endpoint is queried
+        """
+        self.should_check_equality = False
+        return self.do_request('mychannel/thisendpointdoesnotexist123', expected_code=404)
+
```

Comment:
What about a test that does a request to 'mychannel/' ? What happens if the param is missing?